src/authStrategies/LocalAuth.js

```diff
@@ -44,7 +44,8 @@ class LocalAuth extends BaseAuthStrategy {
 
     async logout() {
         if (this.userDataDir) {
-            return (fs.rmSync ? fs.rmSync : fs.rmdirSync).call(this, this.userDataDir, { recursive: true });
+            await this.client.pupBrowser.close();
```

Comment:
Any browser manipulation should be done on the `Client` - could we move this to the client's logout function?

Reply:
@pedroslopez, thanks for the feedback - let me give it a try.

Reply:
@chrishubert this is still not implemented?

Reply:
I've taken the liberty of changing this myself to fix this more quickly.